package.json

```
@@ -8,7 +8,8 @@
     "build": "tsc -b && vite build",
     "preview": "vite preview",
     "lint": "eslint . --ext .ts,.tsx --fix",
-    "format": "prettier --write ."
+    "format": "prettier --write .",
+    "test": "vitest"
```

I prefer this approach better:

```suggestion
    "test": "vitest run",
    "test:watch": "vitest"
```